CONTRIBUTE

```diff
@@ -0,0 +1,109 @@
+HOW TO CONTRIBUTE TO TQDM
+===========================
+
+This file describes how to contribute changes to the project, and how to 
+upload to the pypi repository. 
+
+
+HOW TO COMMIT YOUR CONTRIBUTIONS 
+----------------------------------------------------------- 
+
+Contributions to the project is made using the `Fork & Pull` model. 
+Here's a quickstart on how to do that (your mileage may vary depending 
+on your git config): 
+
+- first create an account on github.
+- then go on the tqdm page `https://github.com/tqdm/tqdm` and fork it
+(click on the fork button).
+- now make a local clone of the project:
+`git clone https://github.com/your_account/tqdm.git`
+- do your changes on your local copy.
+- commit your changes `git commit -m "my message"`
+- TEST YOUR CHANGES (see below).
```

Comment:
The idea I had was not to burden too much the new contributors: first they implement the stuff they want, and after if they can, they try to fix the regressions. But really, ensuring codebase integrity is a job for the main devs who are used to the codebase, for new contributors it can be very overwhelming...